RobotCode/src/main/java/frc/robot/constants/VisionConstants.java

```diff
@@ -16,4 +19,7 @@ public final class VisionConstants
 
     public static final boolean ADJUST_SPEAKER_POSE_FOR_VELOCITY = true;
 
+	public static final Transform3d ROBOT_TO_CAMERA_OFFSET = new Transform3d(new Translation3d(0.5, 0.0, 0.5),
```

Comment:
You used primitive naming convention. It should be RobotToCameraOffset because it's an object.